ironfish/src/sdk.ts

```diff
@@ -157,7 +162,11 @@ export class IronfishSdk {
   }
 
   async node({ databaseName }: { databaseName?: string } = {}): Promise<IronfishNode> {
+    const webSocket = (await require('ws')) as IsomorphicWebSocketConstructor
+    const webRTC = (await require('wrtc')) as IsomorphicWebRtc | undefined
```

Comment:
This variable casing doesn't match that of `WebRtcConnection` and `IsomorphicWebRtc`

Reply:
fixed